src/k2/dto/ControlPlaneOracle.h

```diff
@@ -139,6 +139,18 @@ struct CreateSchemaResponse {
     K2_DEF_FMT(CreateSchemaResponse);
 };
 
+static constexpr int64_t ANY_VERSION = -1;
```

Comment:
Since this is not in a schema-specific namespace/class, you should name it like so: ANY_SCHEMA_VERSION.